### PR TITLE
Add {"_extension_version": MYSQLND_AZURE_VERSION} to connection attr

### PR DESCRIPTION
**Pull request by coko156**
Add a connection attribute "_extension_version" : "mysqlnd_azure-x.x.x", e.x. "_extension_version": "mysqlnd_azure-1.1.1".

`mysqlnd_azure.h: MYSQLND_AZURE_VERSION` variable should be changed if you want to make a new version number.

### How to get this variable:
`select * from performance_schema.session_account_connect_attrs;`
can show what attrs set for current session.

### CODE REVIEW

File: mysqlnd_azure.h [hunk 1]
```diff
@@ -26,6 +26,8 @@
 #include "ext/mysqlnd/mysqlnd.h"
 #include "ext/mysqlnd/mysqlnd_debug.h"
 
+#define MYSQLND_AZURE_VERSION "mysqlnd_azure-1.1.1"
```
Comment (GuuBu) [inline mysqlnd_azure.h:29]: The package.xml need update accordingly too

File: mysqlnd_azure.c [hunk 2]
```diff
@@ -618,6 +618,7 @@ MYSQLND_METHOD(mysqlnd_azure, connect)(MYSQLND * conn_handle,
 
     if (PASS == (*pconn)->m->local_tx_start(*pconn, this_func)) {
         mysqlnd_options4(conn_handle, MYSQL_OPT_CONNECT_ATTR_ADD, "_client_name", "mysqlnd");
+        mysqlnd_options4(conn_handle, MYSQL_OPT_CONNECT_ATTR_ADD, "_extension_version", MYSQLND_AZURE_VERSION);
```
Comment (GuuBu) [inline mysqlnd_azure.c:621]: @hzhang87 You mentioned once we need to expose the driver name in Monlogin. Currently the line above for _client_name is already visible in Monlogin as driver name, should we update this value too? Or Monlogin can also properly parse the field _extension_version?

Comment (coko156) [reply mysqlnd_azure.c:621]: Select * from p_s.session_account_connect_attrs
1. _client_name only give "mysqlnd" , _extension_version gives specific version of mysqlnd_azure. Having discussed with @hzhang87 , we need a method to get extension version number from GW side.
2. GW just show what inside a session attr section but do no parse, from my understand.

**Comment by hzhang87** [comment]
Seems good to me. It will be good if we can have a build and test against GW.

**Review by hzhang87 — APPROVED** [review]

**Review by GuuBu — APPROVED** [review]

File: package.xml [hunk 3]
```diff
@@ -80,6 +80,22 @@
...
+     <api>1.1.1</api>
+     </version>
+     <stability>
+      <release>stable</release>
+      <api>stable</api>
+     </stability>
+     <date>2020-05-13</date>
+     <license uri="http://www.php.net/license">PHP License</license>
+     <notes>
+- 1. New connection attribute "_extension_version", indicate the mysqlnd_azure extension build version.
```
Comment (GuuBu) [inline package.xml:90]: The notes also need update in  section above, where will actually be shown in peck package. Check line 30

Comment (GuuBu) [reply package.xml:90]: @coko156

**Pull request merged**